src/schematic/schematic_json_transformer.py

```diff
@@ -0,0 +1,131 @@
+import json
+from pprint import pprint
+from os.path import dirname
+import copy
+project_root = dirname(dirname(dirname(__file__)))
+from schematic_utils import TRANSFORM_MAP, \
+    make_object, \
+    set_annotation_required, \
+    set_slot_required, \
+    includify_curie, \
+    process_enum_range,\
+    enum_value_object, \
+    schema_value_object, \
+    pascal_to_camel
+
+from linkml_runtime.utils.schemaview import SchemaView
+
+
+# EXAMPLE SCHEMATIC OBJECT
+# {'@id': 'bts:Study',
+#  '@type': 'rdfs:Class',
+#  'rdfs:comment': 'TBD',
+#  'rdfs:label': 'Study',
+#  'rdfs:subClassOf': [{'@id': 'schema:Thing'}],
+#  'schema:isPartOf': {'@id': 'http://schema.biothings.io'},
+#  'sms:displayName': 'Study',
+#  'sms:required': 'sms:true',
+#   "sms:requiresComponent": [
+#     {
+#         "@id": "bts:Study"
+#     },
+#     {
+#         "@id": "bts:DataFile"
+#     }
+# ],
+#  'sms:requiresDependency': [{'@id': 'bts:StudyCode'},
+#                             {'@id': 'bts:StudyName'},
+#                             {'@id': 'bts:Program'},
+#                             {'@id': 'bts:DbGaP'}],
+#  'sms:validationRules': []}
+
+
+class SchematicJSONTransformer(object):
+    def __init__(self, schema_path, output_path):
+        self.schema_path = schema_path
+        self.output_path = output_path
+        self.sv = SchemaView(self.schema_path)
+        self.context = self.generate_context()
+
+        # Build the schematic json
+        self.schematic_classes = []
+        self.class_keys = []
+        self.schematic_properties = []
+        self.prop_keys = []
+        self.generate_schema_object()
+
+    def class_generator(self):
+        for sclass, sdef in self.sv.all_classes().items():
+            class_object = dict()
+            class_object['@type'] = 'rdfs:Class'
+            class_object['@id'] = sdef.definition_uri
+            class_object['rdfs:label'] = sdef.name
```

Comment:
I'm assuming here sdef.name is good enough since it's already CamelCased whereas the pascal_to_camel call at line 91 ensures the slot-to-property ends up being camelCased.